doc/Programming-guide/Key-platform-features/Notifications/index.md

```diff
@@ -10,33 +10,44 @@ sort_idx: 70
 {:toc}
 
 
-The Kaa Notification subsystem enables delivery of messages from the Kaa cluster to endpoints (EP). Unlike configuration data that represents the desired EP state, notifications can be thought of as calls for a dynamic EP action. 
-For example, a notification may cause a client application to display a message on the UI (a user notification).
+The Kaa **Notification subsystem** is used to deliver messages from [Kaa cluster]({{root_url}}Glossary/#kaa-cluster) to [endpoints]({{root_url}}Glossary/#endpoint-ep).
+Unlike configuration data that represents the desired endpoint state, notifications are used as calls for a dynamic endpoint action.
+For example, a notification can cause a [Kaa client]({{root_url}}/#kaa-client) to display a UI message (a user notification).
 
-This guide will familiarize you with the basic concepts of Kaa notifications and programming of the Kaa notification subsystem. It is assumed that you have already set up either a [Kaa Sandbox](http://www.kaaproject.org/download-kaa) or a [full-blown Kaa cluster]({{root_url}}Administration-guide/System-installation/Cluster-setup/) and that you have created at least one [tenant]({{root_url}}Administration-guide/Tenants-and-applications-management/#TODO) and one [application]({{root_url}}Administration-guide/Tenants-and-applications-management/#managing-applications) in Kaa. 
-We also recommend that you review [collecting endpoint profiles guide]({{root_url}}Programming-guide/Key-platform-features/Data-collection) and [endpoint groups]({{root_url}}Programming-guide/Key-platform-features/Endpoint-groups) before you proceed with this guide.
+## Prerequisites
 
-### Basic architecture
+To use the examples below, you need to first set up either a [Kaa Sandbox]({{root_url}}Glossary/#kaa-sandbox) or a full-blown [Kaa cluster]({{root_url}}Glossary/#kaa-cluster).
+After that, you need to create a tenant and an application.
+To do this, you can use the [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/User/createKaaAdmin) or the [Administration UI]({{root_url}}Administration-guide/Users-management/#managing-tenant-admins).
```

Comment:
Wrong name of link [server REST API](should be tenant and user REST API) and  open a wrong page --  creating Kaa admin instead of creating tenant and  tenant admin [ /Programming-guide/Server-REST-APIs/#!/Tenant/editTenant and Programming-guide/Server-REST-APIs/#!/User/editUser]. And Where is API for creating an application?